src/main/java/core/basesyntax/figures/Getarea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.figures;
+
+public interface Getarea {
```

Comment:
Interface names are better when nouns or adjectives. Verbs are used for methods naming
``` suggestion
public interface AreaCalculator {
```